xlators/protocol/client/src/client-rpc-fops.c

```diff
@@ -5098,8 +5045,7 @@ client3_3_lk(call_frame_t *frame, xlator_t *this, void *data)
 
         if ((op_errno == EBADF) && (args->flock->l_type == F_UNLCK) &&
             client_is_setlk(local->cmd)) {
-            client_add_lock_for_recovery(local->fd, args->flock, &local->owner,
-                                         local->cmd);
+            /*Let it pass*/
```

Comment:
client_add_lock_for_recovery() seems to have been added for lock healing which we don't do anyway, so I guess it is okay to remove it.